openapi_python_client/templates/property_templates/union_property.py.jinja

```diff
@@ -24,20 +36,25 @@ def _parse_{{ property.python_name }}(data: Any) -> {{ property.get_type_string(
 {{ property.python_name }} = _parse_{{ property.python_name }}({{ source }})
 {% endmacro %}
 
+{# For now we assume there will be no unions of unions #}
```

Comment:
In the future we should probably be unrolling unions while building the properties. I don't imagine that would be too difficult, just a check on each inner_property to see if it's a UnionProperty.

Reply:
Ah, yeah, that makes sense.